parser/XcodeTestResultParser.php

```diff
@@ -90,7 +90,7 @@ public function parseTestResults($path, $test_results) {
       }
 
       // End of a test.
-      if (!preg_match('/\'-\[.+? test(.+?)\]\' (.+?) \((.+?) seconds\\).$/', $line, $matches)) {
+      if (!preg_match('/\'-\[\S*\s(?:test)*(.+?)\]\' (.+?) \((.+?) seconds\).$/', $line, $matches)) {
```

Comment:
That is really the crux of the change. The test prefix is really optional. 
- If it is not present. No big deal. 
- If it is present we don't want to match on it as you don't really want it in the test name as it is noise.